ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +48,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *rac_description(id object) {
+	if ([object respondsToSelector:@selector(rac_description)]) {
+		return [object rac_description];
+	} else {
+		if (getenv("RAC_DEBUG_SIGNAL_NAMES") != NULL && [object respondsToSelector:@selector(description)]) {
```

Comment:
`-description` is part of [`<NSObject>`](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/1928#discussion_r30458419), which is different from the `NSObject` root class.